subprojects/docs/src/snippets/kotlinDsl/androidBuild/kotlin/build.gradle.kts

```diff
@@ -2,8 +2,8 @@
 plugins {
     id("com.android.application") version "7.3.0" apply false
 // end::android[]
-    kotlin("android") version "1.8.0" apply false
-    kotlin("android.extensions") version "1.8.0" apply false
+    kotlin("android") version "1.8.10" apply false
+    kotlin("android.extensions") version "1.8.10" apply false
```

Comment:
While the `android-extensions` plugin is still published, it is deprecated since [1.4.20](https://kotlinlang.org/docs/whatsnew1420.html#kotlin-android-extensions) and should be removed since [1.8](https://youtrack.jetbrains.com/issue/KT-54013/Report-error-when-using-deprecated-Kotlin-Android-Extensions-compiler-plugin), but not yet.
So I would remove this in the docs. Or in another PR.

Reply:
Thanks @hfhbd. Done in
* https://github.com/gradle/gradle/pull/23775